lib/plugins/aws/package/compile/events/apiGateway/lib/apiKeys.js

```diff
@@ -7,10 +7,11 @@ function createApiKeyResource(that, apiKey) {
   const value = _.isObject(apiKey) && apiKey.value ? apiKey.value : undefined;
   const description = _.isObject(apiKey) ? apiKey.description : undefined;
   const customerId = _.isObject(apiKey) ? apiKey.customerId : undefined;
+  const enabled = _.isObject(apiKey) && !_.isUndefined(apiKey.enabled) ? apiKey.enabled : true;
```

Comment:
I think instead of `_.isUndefined` we can avoid using `lodash` and go with ` apiKey.enabled != null` - what do you think? In this case we will also catch `null` value that we want to ignore